### PR TITLE
Update ramping file with new attach_constraint and create new tables

### LINKED ISSUE #964: Rename, split, create new constraints for ramping-and-unit-commitment.jl
- [x] Create constraint limit_units_on. Copy from units_on variable and update Miro
- [x] Create constraints min_output_... and max_output_... similar to - [ ramping_with_unit_commitment (file ramp...jl). This requires two tables, since they don't have the same indices
- [x] Rename expression units_on, at least inside the file ramp...jl
- [x] Create a new table max_ramp_with_unit_commitment to attach max_ramp_up_... and max_ramp_down...
- [x] Create new table max_ramp_without_unit_commitment to attach related max_ramp constraints. Notice that k = 1 will be an exception case, for now set to 0.0 == 0.0.
- [x] assets_profile_times_capacity_out should be duplicated and attached to corresponding tables

### PR DESCRIPTION
**Pull request by abelsiqueira**
Update ramp file with new attach_constraint, create new tables and change from groupby to loop over the entire table (to use the attach).

Depends on #963

Closes #964

### CODE REVIEW

File: src/constraints/ramping-and-unit-commitment.jl [hunk 1]
```diff
@@ -7,178 +7,217 @@ Adds the ramping constraints for producer and conversion assets where ramping =
...
+                        cons.expressions[:outgoing][row.index-1] ≤
+                        graph[row.asset].max_ramp_up *
+                        row.min_outgoing_flow_duration *
+                        profile_times_capacity[table_name][row.index],
+                        base_name = "max_ramp_up_without_unit_commitment[$(row.asset),$(row.year),$(row.rep_period),$(row.time_block_start):$(row.time_block_end)]"
+                    )
+                end for row in eachrow(cons.indices)
+            ],
+        )
+        #base_name = "max_ramp_down_without_unit_commitment[$(row.asset),$(row.year),$(row.rep_period),$(row.time_block_start):$(row.time_block_end)]"
```
Comment (datejada) [inline src/constraints/ramping-and-unit-commitment.jl:200]:
```suggestion
```

**Review by datejada — APPROVED** [review]
Reviewed through peer programming

File: src/constraints/create.jl [hunk 2]
```diff
@@ -202,10 +213,46 @@ function _create_constraints_tables(connection)
...
+            ON t_high.asset = asset.asset
+        WHERE
+            asset.type in ('producer', 'conversion')
+            AND asset.ramping
+            AND asset.unit_commitment
+            AND asset.unit_commitment_method = 'basic'
         ",
     )
 
+    # TODO: Should there be a `AND NOT asset.unit_commitment` in the WHERE here?
```
Comment (abelsiqueira) [inline src/constraints/create.jl:255]: @datejada, this was a question I had when looking at these tables. The `without_unit_commitment` doesn't check for `asset.unit_commitment = false` or `NOT asset.unit_commitment`

Comment (datejada) [reply src/constraints/create.jl:255]: Good catch! We need to add those checks in the SQL query. Thanks!

**Comment by abelsiqueira** [comment]
Done. I re-requested your review but I'm not working anymore today. If I work before our Monday meeting, I'll just merge

**Pull request merged**